x-pack/plugins/console_extensions/server/lib/spec_definitions/json/generated/text_structure.find_structure.json

```diff
@@ -0,0 +1,32 @@
+{
+  "text_structure.find_structure": {
```

Comment:
Same question here regarding adding body autocomplete for this POST request.

Reply:
Same question for this one.

Reply:
I updated this endpoint to have the same request parameters that correspond with the URL parameters.